### DISCUSSION TITLE
Applies to areas in overviews and scenarios

### DISCUSSION THREAD

**Discussion by salles8863** [microsoft/dynamics365patternspractices#167] [Q&A] [answered]
Hi and thanks for that great "beginning of something big"!  The complete project of the business process catalog is very good and helpful.

I have a question regarding the "apllies to areas": There is only one szenario and one overview which applies to business central. As I am not an expert in BC because I am a senior consultant for F&SCM - but should not every apply to F&SCM also include Busniess Central? 

I tried to figure out my point of view on this:
Acquire to dispose: No BC - has to be added?!
Case to resolution: No BC - has to be added?!
Concept to market: No BC - has to be added?! - In the area related ressources: How to create service items in BC
Design to retire: No BC - has to be added?!
Forecast to plan: No BC - has to be added?!
Hire to retire: No BC - has to be added?!
Inventory to deliver: No BC - has to be added?!
Order to cash: No BC - has to be added?!
Procure to pay: No BC - has to be added?!
Project to profit: No BC - has to be added?!
Prospect to quote: No BC - has to be added?! - Also no F&SCM - It is not the perfect way but you can also build that process in F&SCM
Record to report: No BC - has to be added?!
Service to cash: No BC - has to be added?!

Plan to produce is the only szenario and overwiew which apllies to BC and F&SCM

What is your point of view on this and should I request a change for every single szenario and overview?

Best regards and keep on going ... ;-)

**Comment by edupont04** [comment, marked as answer]
Thank you for your question! Most high-level processes are described in a way that it doesn't really matter which Dynamics 365 app is the optimal one to use for that according to Microsoft. Sometimes, you would want to use more than one app plus some customizations or ISV extensions, for example. So the list of "applies to" is not intended to be 100% accurate but a "ball park shot" more than anything.
Dynamics 365 Business Central is often not listed as supporting a certain end-to-end scenario because we didn't have a subject matter expert available at the time of publishing for verifying which processes are supported out-of-the-box for Business Central. We publish updates every two weeks, and Business Central will be added if and when we can confirm support.

I hope this answers your concerns?

**Comment by rachel-profitt** [comment]
If you would like to help contribute to add more BC processes/patterns to the catalog, you can put those requests in using the Issues tab in GitHub. We are actively seeking more BC experts to help us build out the patterns for Business Central and we think this is a great opportunity for partners and consultants to showcase their expertise.
